packages/realm-server/scripts/start-all.sh

```diff
@@ -2,7 +2,7 @@
 
 NODE_NO_WARNINGS=1 start-server-and-test \
   'run-p start:pg start:matrix start:smtp start:development start:base:root' \
-  'http-get://localhost:4201/base/fields/boolean-field?acceptHeader=application%2Fvnd.card%2Bjson|http-get://localhost:4203/fields/boolean-field?acceptHeader=application%2Fvnd.card%2Bjson|http-get://localhost:4201/drafts/index?acceptHeader=application%2Fvnd.card%2Bjson|http://localhost:8008|http://localhost:5001' \
+  'http-get://localhost:4201/base/fields/boolean-field?acceptHeader=application%2Fvnd.card%2Bjson|http-get://localhost:4203/fields/boolean-field?acceptHeader=application%2Fvnd.card%2Bjson|http-get://localhost:4201/drafts/_readiness-check??acceptHeader=application%2Fvnd.api%2Bjson|http://localhost:8008|http://localhost:5001' \
```

Comment:
I added the `readiness-check` endpoint to address the problem I had during testing this feature using realms in our development setup. If I configured the drafts realm to be permissioned in a way where it only allows reading by a specific user, then the request to /drafts/index would fail with 403 permission denied and the start:all command would fail to run the realms.